submissions/SerhiiYakovenko/js-dom/script.js

```diff
@@ -0,0 +1,128 @@
+const items = [
+  {
+    heading: "City at day",
+    img: "img/city.jpg",
+    text:
+      "Enter the massive open world of Night City, a place that sets new standards in terms of visuals, complexity and depth.",
+  },
+  {
+    heading: "City at night",
+    img: "img/night-city.jpg",
+    text: `Cyberpunk 2077 is an open-world, action-adventure story set in Night City, a megalopolis obsessed with power, glamour and body modification.
+        You play as V, a mercenary outlaw going after a one-of-a-kind implant that is the key to immortality.
+        You can customize your character’s cyberware, skillset and playstyle, and explore a vast city where the choices you make shape the story and the world around you.`,
+  },
+  {
+    heading: "Mercenary",
+    img: "img/mercenary.jpg",
+    text:
+      "Become a cyberpunk, an urban mercenary equipped with cybernetic enhancements and build your legend on the streets of Night City.",
+  },
+  {
+    heading: "Implants",
+    img: "img/implant.jpg",
+    text:
+      "Take the riskiest job of your life and go after a prototype implant that is the key to immortality.",
+  },
+];
+
+const body = document.querySelector("body");
+
+// Header
```

Comment:
it's a redundant comment. it's clear what you do in your further operation. they are obvious.